ITE/10/README.adoc

```diff
@@ -0,0 +1,265 @@
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 10
+
+| Title
+| Layouts for in-toto Attestations
+
+| Sponsor
+| link:https://github.com/adityasaky[Aditya Sirish A Yelgundhalli]
+
+| Status
+| Draft
+
+| Type
+| Standards
+
+| Created
+| 2023-01-07
+
+|===
+
+[[abstract]]
+== Abstract
+
+A previous in-toto Enhancement, ITE-6, presented the in-toto Attestation
+Framework. This framework introduced the idea of context-specific attestations
+with their own definitions. This ITE builds on ITE-6 and proposes several
+modifications to in-toto layouts that allow for defining and verifying policies
+over ITE-6 attestations.
+
+[[specification]]
+== Specification
+
+Note: this document assumes the reader is familiar with ITE-6 and the in-toto
+Attestation Framework.
+
+in-toto layouts have two key mechanisms for specifying policies: steps and
+inspections. Steps are used to reason about and express constraints regarding
+operations performed as part of the supply chain. Constraints are currently
+expressed using in-toto artifact rules against the materials and products of a
+step. Each step definition also declares the set of functionaries authorized to
+perform the step and the required number of functionaries that need to sign off
+on the step.
+
+Inspections, on the other hand, are executed during the verification workflow to
+complement in-toto's step verifications with extra checks. Typically,
+inspections invoke external commands or scripts shipped with the layout. They
+may also use artifact rules. Currently, inspections are used for constraints
+in-toto layouts do not support. For example, in-toto links can record arbitrary
+information in their opaque environment and byproducts fields. To validate their
+contents, supply chain owners must rely on custom verification scripts invoked
+as inspections.
+
+=== Classifying Predicates
+
+Prior to ITE-6, all steps were validated against in-toto links. Inspection
+executions also resulted in the generation of in-toto links that were then
+validated. With the introduction of the in-toto Attestation Framework, it is
+necessary to rework the layout to support verifying a variety of _predicates_.
+As the framework is designed to generate claims pertaining to how software is
+produced, each predicate type can be mapped to a step in the software supply
+chain. However, it is first necessary to understand the nature of each step, and
+therefore its predicate type. in-toto was originally designed with the
+assumption that a step in the software supply chain affects in some manner the
+artifacts that constitute the supply chain. ITE-6 showed that some steps do not
+fit the materials and products model. This is a factor in _verifying_ such
+predicates using in-toto layouts.
+
+Therefore, all predicates supported for verification in an in-toto layout must
+categorized as one of two types:
+
+* the link type: these are based on classic in-toto links for steps that affect
```

Comment:
I wonder if we can be a little more precise: instead of saying "affect", could we say "transform" or "mutate"?